Tests iOS/MyList/SearchTests.swift

```diff
@@ -329,8 +333,8 @@ class SearchTests: XCTestCase {
     }
 
     // MARK: - Search Error State
-    func test_search_showsErrorView_andReportView() {
-        server.routes.post("/graphql") { request, eventLoop -> FutureResponse in
+    func test_search_showsErrorView() {
```

Comment:
As per slack conversation, reverted test to not check for the report an issue feature. An investigation ticket will be created to follow up.
https://pocket.slack.com/archives/C01UH57EJKD/p1686592242193829?thread_ts=1686588403.349329&cid=C01UH57EJKD

Reply:
👍